concepts/58 jQuery Components/05 Add DevExtreme to a jQuery Application/10 Local Files.md

```diff
@@ -1,19 +1,25 @@
 You can find all the required files in the <a target="_blank" onclick="trackGAEvent('Download Trial', 'Complete zip', 'jQuery article');" href="https://go.devexpress.com/DevExpressDownload_DevExtremeCompleteTrialZip.aspx">DevExtreme ZIP archive</a> or in DevExtreme folder (%ProgramFiles(x86)%\DevExpress 20.1\DevExtreme\Sources) if you used the <a target="_blank" onclick="trackGAEvent('Download Trial', 'Complete', 'jQuery article');" href="https://go.devexpress.com/DevExpressDownload_DevExtremeCompleteTrial.aspx">Windows installer</a>. Copy the Lib folder into the folder with your application. Then, link jQuery and DevExtreme stylesheets and scripts in the `<head>` tag on your index page in the following order:
```

Comment:
Change "all the required files" to "all required files"
Change "in DevExtreme folder" to "in the DevExtreme folder"
Change the last sentence to:
Then, link jQuery and DevExtreme stylesheets/scripts in the index page's `<head>` tag in the following order: